hucache.http/Cache.fs

```diff
@@ -5,10 +5,10 @@ open hucache.http.types
 open hucache.http.github
 
 let lookup (key: IssueKey) : FullPayload option =
-    match getFromDb (key) with
+    match loadIssue key with
     | Some payload -> Some(payload)
     | None -> 
         let issue = fetchIssue key
         match issue with 
-        | Some i -> Some((storeInDb (key, i)))
+        | Some i -> Some((storeIssue (key, i)))
         | None -> None
```

Comment:
Check out `bind` and `map`: 
https://msdn.microsoft.com/visualfsharpdocs/conceptual/core.option-module-%5bfsharp%5d